src/lib/errors/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './ErrorFactory';
+export * from './errorHandler';
```

Comment:
This is probably unrelated to the review, but why is it necessary to export it from this index file?

Reply:
It's only about export (https://github.com/DXHeroes/dx-scanner/pull/305/files#diff-f41e9d04a45c83f3b6f6e630f10117feR10) from index file directly instead of a specific file.

Reply:
Oh I see, I must have overlooked it. Thanks for the explanation :)